bin/CMakeLists.txt

```diff
@@ -41,6 +41,22 @@ configure_file(${CMAKE_CURRENT_SOURCE_DIR}/nrnivmodl_makefile_cmake.in
 
 # if running from the build folder (e.g. make test) may need this.
 file(COPY ${CMAKE_CURRENT_SOURCE_DIR}/nrnpyenv.sh DESTINATION ${PROJECT_BINARY_DIR}/bin)
+# Make sure nrnivmodl is executable in the build folder, so we can execute it to prepare test files.
+# This can be done more elegantly in newer CMake versions; v3.19+ have file(CHMOD ...) and v3.20+
+# support setting permissions directly in configure_file(...).
+file(
+  COPY ${PROJECT_BINARY_DIR}/bin/nrnivmodl
+  DESTINATION ${PROJECT_BINARY_DIR}/bin/tmp
+  FILE_PERMISSIONS
+    OWNER_READ
+    OWNER_WRITE
+    OWNER_EXECUTE
+    GROUP_READ
+    GROUP_EXECUTE
+    WORLD_READ
+    WORLD_EXECUTE)
+file(RENAME ${PROJECT_BINARY_DIR}/bin/tmp/nrnivmodl ${PROJECT_BINARY_DIR}/bin/nrnivmodl)
+file(REMOVE ${PROJECT_BINARY_DIR}/bin/tmp)
```

Comment:
Good point. Actually I think it needs to be `nrn_configure_dest_src(nrnivmodl bin/tmp nrnivmodl bin)` because the source file is in `{source_dir}/bin`, but I've changed it to that in eefa2d9f9a3aea89d48e6530ebebb7956f851183.